example/screens/MessageCenterScreen.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Sample React Native App
+ *
+ * HomeScreen: Contains only the channelId for the moment.
+ */
+'use strict';
+
+import {
+ UrbanAirship,
+} from 'urbanairship-react-native'
+
+import React, {
+  Component,
+} from 'react';
+
+import {
+  Text,
+  View,
+  Image,
+  ScrollView,
+  Button,
+} from 'react-native';
+
+import styles from './../Styles'
+
+export default class MessageCenterScreen extends React.Component {
+
+  constructor(props) {
+    super (props);
+
+    this.handleMessageCenterDisplay = this.handleMessageCenterDisplay.bind(this);
+  }
+
+  componentDidMount() {
+    this.handleMessageCenterDisplay()
+  }
+
+  handleMessageCenterDisplay() {
+    UrbanAirship.displayMessageCenter()
+  }
+
+
+  render() {
```

Comment:
The ticket must not of been clear, you need to show a message center using react native components not just a button to show the native message center.

Should be two screens:
- Listing of messages using React Native ListView
- A message using the UAMessageView